### PR TITLE
Code assignment 2

### PR DESCRIPTION
**Pull request by ibrohimkhan**

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "magink",
   "version": "0.1.0",
-  "homepage": "https://swanky-dapps.github.io/magink-dapp",
+  "homepage": "https://github.com/ibrohimkhan/magink-dapp",
```
Comment (Maar-io) [inline frontend/package.json]: this should be
`https://yaymalaga.github.io/magink-dapp/`

and please add this link to Repo description

**Review by Maar-io — APPROVED** [review]

**Comment by Maar-io** [comment]
The text for minting should be changed. The one that starts with "Darling..."

**Comment by Maar-io** [comment]
Where do I see the minted NFT?
Please add in the README or similar the contract address and how to see the NFT.
Think of all of your friends who will want to see it :)

**Comment by Maar-io** [comment]
I wanted to check your contract but did not find metadata and I can't interact. Please add metadata file for the contract

**Comment by ibrohimkhan** [comment]
Hey, thanks for your responses! Appreciate it!
metadata.json file is here: 
https://github.com/ibrohimkhan/magink-dapp/blob/code-assignment-2/frontend/src/metadata.json

and I have changed the contract address here:
https://github.com/ibrohimkhan/magink-dapp/blob/code-assignment-2/frontend/src/const/index.ts

**Comment by Maar-io** [comment]
Great job, I hope you did learn some new tricks with this task, would love to hear how was the experience and what would you do differently if you were giving the task?
Was it hard?
Did e2e test work for you?

**Comment by ibrohimkhan** [comment]
Hey @Maar-io, thanks a lot! It was great experience for me to learn and practice smart contract development using ink!, openbrush and swanky. The hard part is UI for me since I don't have much experience in React and Typescript. But I am interested to grab the basics soon. **e2e tests passed on my local machine** but I could not configure my github actions to pass them there, so stoped with unit testing scripts. Definitely I'll continue to work on frontend part in order to build completed dapp.

**Comment by Maar-io** [comment]
Thank you! Good job

**Pull request merged**